app/videobridge/server/settings.js

```diff
@@ -89,6 +89,15 @@ Meteor.startup(function() {
 				public: true,
 			});
 
+			this.add('Jitsi_URL_Room_Suffix', '', {
+				type: 'string',
+				enableQuery: {
+					_id: 'Jitsi_Enabled',
+					value: true,
+				},
```

Comment:
```suggestion
				},
				i18nLabel: 'URL_room_suffix',
```
Since you are not adding the i18n string with the name of the setting this i18nLabel will be required, sorry to suggest to remove it earlier.